README.md

```diff
@@ -214,13 +214,32 @@ Where the prefix is either lowercase `http` or uppercase `HTTP` and the username
 
 For example: `http://username\@mail.foo:p\@ssw\@rd@www.foo.org:100` where username is `username@mail.foo`, password is `p@ssw@rd`, the proxy server is `www.foo.org` and the port is 100.
 
-The environment variable needs to be set for the root user account. For example, for a fictive proxy server, user and password, the environment variable `http_proxy` can be set for the root user with:
+The environment variable needs to be set for the root user account. For example, for a fictive proxy server, user and password, the environment variable `http_proxy` can be set for the root user manually via console with:
 
 ```
 sudo -E su
 export http_proxy=http://user:password@wwww.foo.org:100//
 ```
 
+The environment variable can also be set in the OSConfig service unit file by uncommenting and editing the following line in [src/agents/pnp/daemon/osconfig.service](src/agents/pnp/daemon/osconfig.service):
+
+```
+# Uncomment and edit next line to configure OSConfig with a proxy to connect to the IoT Hub
+# Environment="http_proxy=http://user:password@wwww.foo.org:100//"
+```
+
+After editing the service unit file, stop and disable osconfig.service, rebuild OSConfig, then enable and start osconfig.service:
+
+```bash
+sudo systemctl stop osconfig.service
+sudo systemctl disable osconfig.service
+cd build
+cmake ../src -DCMAKE_BUILD_TYPE=Release|Debug -Duse_prov_client=ON -Dhsm_type_symm_key=ON -DBUILD_TESTS=ON|OFF
+cmake --build . --config Release|Debug  --target install

```

Comment:
Is building again needed? I believe not.

Reply:
Yes, it is. That binplaces all necessary files, including this file.